packages/convertkit/src/forms/index.tsx

```diff
@@ -1,69 +1,91 @@
 import * as React from 'react'
 
-//TODO: separate the styling and the behavior so that it can be customized
-
 const FORM_ID = process.env.NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM
 
 type ConvertkitSubscribeFormProps = {
   onSubmit?: () => void
+  button?: React.DetailedHTMLProps<
+    React.ButtonHTMLAttributes<HTMLButtonElement>,
+    HTMLButtonElement
+  >
+  /** text below submit button */
+  comment?: string
+  classNames?: {
+    input?: string
+    label?: string
+    button?: string
+    /** classNames for text below submit button */
+    comment?: string
+    /** children container classNames */
+    children?: string
+    form?: string
+    /** required field indicator (*) classNames */
+    asterisk?: string
+    buttonContainer?: string
+  }
 }
 
 const ConvertkitSubscribeForm: React.FC<ConvertkitSubscribeFormProps> = ({
   children,
   onSubmit,
+  button,
+  comment,
+  classNames = {
+    input:
+      'w-full rounded-sm dark:bg-gray-900 dark:text-white dark:placeholder-gray-300',
+    label: 'text-sm font-semibold block',
+    button:
+      'w-full px-3 py-2 font-semibold dark:bg-white dark:text-black bg-black text-white rounded-sm',
+    comment: 'text-xs italic text-center',
+    form: 'space-y-4 w-full',
+    children: 'pb-4 font-bold sm:text-xl text-lg tracking-tight',
+    asterisk: 'opacity-50',
+    buttonContainer: '',
+  },
```

Comment:
these are the defaults.

Reply:
i think this should be merged with incoming values and not either/or